src/styles/_catalog.scss

```
@@ -1,10 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  gap: 48px;
   max-width: 944px;
   margin: 0 auto;
 
+  @media(min-width: 488px) {
+    grid-template-columns: repeat(2, 1fr);
```

```suggestion
    grid-template-columns: repeat(2, $card-width);
```
you can use card-width here